platforms/documentation/docs/src/snippets/bestPractices/avoidDuplicateDependencies/tests/avoidDuplicateDependencies.avoid.out

```diff
@@ -0,0 +1,67 @@
+
+> Task :avoid-this:dependencies
+
+------------------------------------------------------------
+Project ':avoid-this'
+------------------------------------------------------------
+
+annotationProcessor - Annotation processors and their dependencies for source set 'main'.
+No dependencies
+
+api - API dependencies for the 'main' feature. (n)
+\--- org.jetbrains:kotlinx-coroutines-core:1.10.0 (n)
+
+apiElements - API elements for the 'main' feature. (n)
+No dependencies
+
+compileClasspath - Compile classpath for source set 'main'.
+\--- org.jetbrains:kotlinx-coroutines-core:1.10.0 FAILED
```

Comment:
❓ Why are there failures here?

Reply:
Hm, for some reason I missed it. Let me double check, I thought I forgot to add mavenCentral repo, but it seems issue in something else.

Reply:
Ok, it was awful mistake of mine. I messed up with corutines coordinates :/ 
fixed now.